mycroft/configuration/mycroft.conf

```diff
@@ -95,8 +95,6 @@
   // General skill values
   "skills": {
     "msm": {
-      // Relative to "data_dir"
-      "directory": "skills",
```

Comment:
we should support this at least during next major version, i personally use a changed .conf to set the data_dir variable and expect my skills there. My use case is having a dev environment with different skills. If they all come from xdg directories i lose this isolation

Reply:
to expand on this, the default data_dir in .conf should be changed to the correct xdg location

just like this fedora packager did https://gitlab.com/LyesSaadi/spec/-/blob/master/mycroft/mycroft/mycroft.conf

Reply:
> the default data_dir in .conf should be changed to the correct xdg location

There is no such thing though. The "correct" XDG location is either `$XDG_DATA_HOME/mycroft/skills` if `$XDG_DATA_HOME` exists, and otherwise `~/.local/share/mycroft/skills`. The whole point of the specification is that you can't assume `~/.local/share`.

Reply:
you can assume whatever place msm is installing things into, its just a default, as i said the point is allowing users to override it / be backwards compatible

Reply:
oh i think i misunderstood, probably the best way is to support a unique string, eg, `data_dir: "{XDG}"` which would be resolved by the code itself